### ISSUE TITLE
Quotes are stripped out of arrays, possibly only in GraphML imports.

### ISSUE THREAD

**Issue by tetious** [ArcadeData/arcadedb#122]
### OS: docker

For this query:
`MATCH (m:Movie{title:'The Polar Express'})<-[e:ACTED_IN]-(v:Person) RETURN e, v`

## Expected behavior
The edge property 'roles' should be a properly formed array, as is shown here from Neo4J's view of the import:
![Screen Shot 2021-10-10 at 20 31 13](https://user-images.githubusercontent.com/55581/136718613-ac6173e6-dc62-4114-8656-f8199f9fade4.png)

## Actual behavior
The quotes are missing, making the array unparsable:
`"roles": "[Hero Boy, Father, Conductor, Hobo, Scrooge, Santa Claus]"`

I've confirmed arrays can be created and returned successfully via parameterized inserts, so I suspect this is a bug in the import code. Could be wrong, though!

**Comment by lvca** [comment]
@antarcticgiraffe could you please take this when you can? Thanks!

**Comment by lvca** [comment]
@tetious you're right, if I import the database from scratch I have the same issue. Checking into it. Thanks for the report.
